examples/02-update_model/CMakeLists.txt

```diff
@@ -11,4 +11,4 @@ TARGET_LINK_LIBRARIES (${EXAMPLE_NAME}
 # NOTE: this requires an install build
 SET(RESOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/resources)
 FILE(MAKE_DIRECTORY ${RESOURCE_DIR})
-FILE(COPY rprbot.urdf DESTINATION ${RESOURCE_DIR})
+FILE(COPY rprbot.urdf toro.urdf DESTINATION ${RESOURCE_DIR})
```

Comment:
Please do not share the toro urdf model file publicly. They asked us not to do it at DLR. Can you make examples that do not depend on it ?

Reply:
Switched to a simpler parallel robot.